src/Controller/ClassController.php

```diff
@@ -39,6 +41,9 @@ class ClassController extends UserAwareController
     /* @var string $classificationDisplayMode */
     protected $classificationDisplayMode;
 
+    /* @var bool $classificationGroupedDisplay */
+    protected $classificationGroupedDisplay;
```

Comment:
```suggestion
    protected bool $classificationGroupedDisplay;
```
It should be fine to use native types for new properties.